connector_prestashop_catalog_manager/models/__init__.py

```diff
@@ -0,0 +1,10 @@
+# © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from . import binding
+from . import product_category
+from . import product_product
+from . import product_template
+from . import product_image
+
+# from . import ir_translation
```

Comment:
I think it would be nice to have a TODO comment directly if you commenting this out temporary